src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/PSUserAgent.cs

```diff
@@ -123,15 +124,23 @@ internal static string Platform
         {
             get
             {
-                if(RuntimeInformation.IsOSPlatform(OSPlatform.Windows)){
+                if(System.Management.Automation.Platform.IsWindows)
+                {
                     return "Windows NT";
-                }else if(RuntimeInformation.IsOSPlatform(OSPlatform.OSX)){
+                }
+                else if(System.Management.Automation.Platform.IsMacOS)
```

Comment:
Above I suggest rename _internal_ Platform property to exclude using "System.Management.Automation".

Reply:
any preference on the name? how about OSPlatform? @iSazonov

Reply:
It is a string name - maybe PlatformName?